frontend/src/components/pages/AdminPage.tsx

```diff
@@ -44,7 +45,7 @@ const ErrorState = (): React.ReactElement => (
   </VStack>
 );
 
-const getAdminUser = (user: AdminUser) => {
+const getUser = (user: User) => {
```

Comment:
I changed this function to getUsers instead of creating a new getTeacherUsers function, since I thought they would do the same thing.